rhv_checks.py

```diff
@@ -203,11 +203,32 @@ def check_datacenters_status(system, **kwargs):
         print("Ok: all datacenter(s) are in the OK state: {}".format(okay))
         sys.exit(0)
 
+
+def check_storage_domain_attached_status(system, **kwargs):
+    """ Check the usage of all the datastores on the host. """
+    okay, critical, all = [], [], []
+    storage_domains_service = system.api.system_service().storage_domains_service()
+    all_sd = storage_domains_service.list()
+    data_centers_service = system.api.system_service().data_centers_service()
+    all_dc = data_centers_service.list()
+
+    for dc in all_dc:
+        print(dc.id)
+        dc_service = data_centers_service.data_center_service(dc.id)
+        attached_sds_service = dc_service.storage_domains_service()
```

Comment:
is there a way to write line 217+218 as only one line?

Reply:
For simplicity I have kept these in 2 separate sentences, it will be easier to get `dc_service` for each `dc.id`.